spec/lib/lorekeeper/json_logger_spec.rb

```diff
@@ -286,6 +286,17 @@
           logger.write(message)
           expect(io.received_message).to eq(message)
         end
+
+        context 'non-representable data' do
+          let(:message) { Float::NAN }
```

Comment:
To check the serialization of a symbol key, can we use a JSON object here?

```suggestion
          let(:message) { { message: Float::NAN } }
```

Reply:
Also, maybe it's good to add a test case to cover `rescue => e` part

https://github.com/JordiPolo/lorekeeper/blob/20bb011bbd3fe08f8599da5c17e64130be1fde08/lib/lorekeeper/json_logger.rb#L104-L105